configuration.ml

```diff
@@ -72,14 +57,22 @@ let of_json_from_http json_dict now =
         else Ok ()
       in
       let* server_ip = Ipaddr.of_string server_ip in
-      Ok { certificate; private_key; server_ip; server_port; updated_at = now }
+      Ok
+        {
+          name;
```

Comment:
there's no validation/restriction done for "name" -- is this intentional? in the javascript, it is checked with "is_valid_name", but of course someone could just not execute the javascript. I guess we need to do validation (size and content).